Sources/GradientView.swift

```diff
@@ -25,6 +25,7 @@
 
 import Foundation
 
+// swiftlint:disable identifier_name
```

Comment:
this way, all these warnings are disable. Please re-enable it after it is not needed or use single line comment: `swiftlint:disable:this identifier_name`